### PR TITLE
Remove GTSpotterHelp

### LINKED ISSUE #9074: Remove GTSpotterHelp
- based on #9047 we should remove help books of GT
- now lets remove the GTSpotterHelp

### PR DESCRIPTION
**Pull request by astares**
Fix #9074

### CODE REVIEW

File: src/GT-Spotter-UI/GTSpotterHeaderBrick.class.st [hunk 1]
```diff
@@ -53,12 +52,7 @@ GTSpotterHeaderBrick >> helpButton: aBrick [
 	
 	helpButton
 		hintTitle: 'Spotter help';
-		action: [ 
-			self spotterModel exitDueTo: #actionButton.
-			tree := HelpBrowser open instVarNamed: #treeMorph.
-			item := tree allItems detect: [:each | 
-				each complexContents item key = GTSpotterHelp key].
-			tree selection: item ].
+		action: [ self inform: 'GT Help is not available anymore' ].
```
Comment (gcotelli) [inline src/GT-Spotter-UI/GTSpotterHeaderBrick.class.st]: Why keeping a button that does nothing but showing that something is not available? Isn't it better to just remove the button from the UI?

Comment (estebanlm) [reply src/GT-Spotter-UI/GTSpotterHeaderBrick.class.st]: yeah, I agree with this, @astares :)

**Comment by astares** [comment]
Sorry - havent seen the comment. Will take care

**Comment by Ducasse** [comment]
I wanted to integrate it. 
But I do not get why we get classSide instead of class

```

 { #category : #'*Calypso-SystemPlugins-Reflectivity-Browser' }
 ClyQueryBrowserMorph classSide >> browseBreakpoints [
 	"Excludes methods that contain halts marked as for testing. To browse all breakpoints, see this class's browseBreakpointsIncludingHaltsForTesting method"
 	| query |
 	query := ClyActiveBreakpointsQuery as: (ClySortedQueryResult using: ClySortMethodByPackageFunction ascending).
 	^ClyQueryBrowserMorph openOn: query
 ]
```

**Comment by astares** [comment]
Dont now where it comes from - it was saved by Iceberg that way. Changed from classSide to class now manually

**Comment by estebanlm** [comment]
Mmm... in fact I am on the way to remove the whole GTSpotter (and the help with it). 
I think I will close this and wait for the complete removal, it you do not mind :P

**Pull request closed without merging**